File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_shopping_cart_item_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusShoppingCartItemForm:
    """Shopping cart item creation form"""
    cart_id: int
    cart_group_uuid: str
    product_id: int
    sku_id: int
    quantity: int
    price: float
    selected: bool

File: sdkwork-backend-sdk-typescript/src/types/plus-shopping-cart-item-form.ts
/** Shopping cart item creation form */
export interface PlusShoppingCartItemForm {
  /** çî å¢¿æî¯D */
  cartId: number;
  /** çî å¢¿æï¹åç¼åUID */
  cartGroupUuid: string;
  /** éåæ§ID */
  productId: number;
  /** SKU ID */
  skuId: number;
  /** éåæ§éä¼´åº */
  quantity: number;
  /** P ri ce */
  price: number;
  /** éîæé«å¤è */
  selected: boolean;
}
